src/pages/student/quizzes/QuizResult/QuizAnswerResult/components/MultipleChoiceType/index.js

```diff
@@ -41,20 +41,21 @@ const MultipleChoiceType = ({ question, answer }) => {
   return (
     <Fragment>
       <div className={style.question}>
-        <p className={style.paragraph}>
+        <p>
           {' '}
           {question.id}. {question.question}{' '}
         </p>
         {question?.choices.map((choice, idx) => (
           <Card key={idx} className={style.cardbody1}>
```

Comment:
- [ ] Create a method for styling to be passed on classname. If the answered item from multiple choice is correct, set border color to green, if not, set to red.